src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -726,6 +726,11 @@ module Types = struct
             ~args:Arg.[]
             ~resolve:(fun _ {With_hash.hash; _} ->
               Stringable.State_hash.to_base58_check hash )
+        ; field "stateHashField" ~typ:(non_null string)
+            ~doc:"Bigint field-element representation of stateHash"
```

Comment:
According to the example in the PR description this should be base58check. Did you change your mind? base58 is the most useful representation here right?

Reply:
We decided to keep both, base58check is useful as that’s what we use within the daemon and display, but the raw number exposed means we don’t need a base58 decoder in JavaScript right now

Reply:
We still have the other stateHash, right John?

Reply:
yes

Reply:
hmm https://www.npmjs.com/package/base58check